README.md

```diff
@@ -1,13 +1,116 @@
 # 💪 strong-config
 
+<!-- markdownlint-disable line-length -->
+[![pipeline status](https://git.brickblock.sh/devops/strong-config-ts-mirror/badges/master/pipeline.svg)](https://git.brickblock.sh/devops/strong-config-ts-mirror/commits/master) [![coverage report](https://git.brickblock.sh/devops/strong-config-ts-mirror/badges/master/coverage.svg)](https://git.brickblock.sh/devops/strong-config-ts-mirror/commits/master)
+<!-- markdownlint-enable line-length -->
+
 Secret management for NodeJS. Based on [SOPS](https://github.com/mozilla/sops).
 
-[![pipeline status](https://git.brickblock.sh/devops/strong-config-ts-mirror/badges/master/pipeline.svg)](https://git.brickblock.sh/devops/strong-config-ts-mirror/commits/master)
+Have you ever pushed secrets to your git remote by accidentially committing
+the `.env` file? Then `strong-config` (built by Brickblock) is here to help:
 
-[![coverage report](https://git.brickblock.sh/devops/strong-config-ts-mirror/badges/master/coverage.svg)](https://git.brickblock.sh/devops/strong-config-ts-mirror/commits/master)
+- Manage configs for multiple runtime environments, e.g. different clusters
+- Encrypt secrets at rest and automatically decrypt them at runtime
+- Validate configs against a custom schema
+- For Typescript users: automatically generate types based on schema
 
 ## How-to
 
-Set the environment variable `RUNTIME_ENVIRONMENT`
+1. Install `strong-config` in your project
+
+    ```sh
+    npm install @strong-config/node
+    yarn add @strong-config/node
+    ```
+
+1. Define runtime environment and create first config file
+
+    `strong-config` uses an environment variable to determine which config file
+    to load. By default, this is `RUNTIME_ENVIRONMENT`.
+
+    ```sh
+    export RUNTIME_ENVIRONMENT=development # or similar, depending on your shell
```

Comment:
I don't quite understand why we need this line here, what am I missing?

Reply:
Will soon be `NODE_ENV` instead but why wouldn't we need it? Gotta tell strong-config which config to load.

I wouldn't set `NODE_ENV=development` implicitly so the user isn't aware. Shouldn't mess with the user's env vars

Reply:
hmm, but why would this need to be set in the context of creating a new folder and a new file?

`mkdir config` and `echo 'blub' > config/development.yaml` both don't access `NODE_ENV` in any way or am I missing something?

Reply:
Are are you referring to setting `NODE_ENV` in a separate point?

I'm not opposed to the current draft because it shows the mapping between file name and `NODE_ENV`. In this light, both steps (setting NODE_ENV, and, creating a dir+file) fit together

Reply:
Yeah, my main point is that to me it feels the wrong place to set the env var because it isn't needed for this. To me as an engineer i would ask myself: "Why would I need to set an env var for creating a folder?"

IMO we should set it once we actually use it

Reply:
Since you have a very clear how this README should look like, where should that step go? It must be fore the JS block.

Just gonna ask to save us both some time